pyteal/compiler/compiler_test.py

```diff
@@ -1,7 +1,10 @@
+from pathlib import Path
 import pytest
 
 import pyteal as pt
 
+FIXTURES = Path.cwd() / "tests" / "teal" / "router"
```

Comment:
Very difficult to read diff. In summary, I'm moving all the teal code in `test_router_app` into fixtures in the directory `tests/teal/router`. I'm doing so as to also comply with usage in #650

Reply:
nit: rename to `ROUTER_FIXTURES`